### PR TITLE
Migration specifications du contenu

### PR DESCRIPTION
**Pull request by prycrane**
Pull request for moving Content and Information Architecture Specification section from AEM to design-system.

design-system: https://github.com/canada-ca/design-system/pull/200

(File numbers may not match English numbers for this round.  Will get repaired as the migration continues.)

### CODE REVIEW

**Comment by prycrane** [comment]
Perhaps we can meet this afternoon and flip this into production.

File: architecture/utilisation-concept-canadaca.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+altLangPage: "https://design.canada.ca/architecture/usage-canadaca-design.html"
+breadcrumbs:
+  - title: À propos de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos.html"
+  - title: Système de conception de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html"
+  - title: Spécifications du contenu et de l'architecture de l'information pour Canada.ca
+    link: "{{ site.url }}/architecture/specifications-contenu-architecture-information-canada.html"
```
Comment (delisma) [inline architecture/utilisation-concept-canadaca.md:9]: Use of variable has to be tested.

File: architecture/utilisation-concept-canadaca.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+date: 2017-10-05
+dateModified: 2023-03-28
+description: "Les exigences énumérées dans ce document s’appliquent aux ministères et aux autres parties de l’administration publique établie comme l’établissent les annexes I, I.1 et II de la Loi sur la gestion finances. Par conséquent, les institutions de la portée doivent appliquer les exigences en matière de conception de Canada.ca pour tous les sites Web ou toutes les services numériques."
+layout: default
+title: "Qui doit utiliser le sysème de conception de Canada.ca"
+---
+<p class="gc-byline"><strong>De : <a href="/fr/secretariat-conseil-tresor.html">Secrétariat du Conseil du Trésor du Canada</a></strong></p>
```
Comment (delisma) [inline architecture/utilisation-concept-canadaca.md:16]:
```suggestion
<p class="gc-byline"><strong>De : <a href="https://www.canada.ca/fr/secretariat-conseil-tresor.html">Secrétariat du Conseil du Trésor du Canada</a></strong></p>
```

File: configurations-conception-communes/widget-medias-sociaux.html
Comment (delisma) [inline configurations-conception-communes/widget-medias-sociaux.html:1]: Revert changes to remove this file from the PR

File: modeles-obligatoire/pages-publics-cibles.html
Comment (delisma) [inline modeles-obligatoire/pages-publics-cibles.html:1]: Revert changes to remove page from PR

File: modeles-obligatoire/pages-theme-backup.html
Comment (delisma) [inline modeles-obligatoire/pages-theme-backup.html:1]: Should be deleted. Will file issue in the future.

File: modeles-obligatoire/theme-sujet-draft.html
Comment (delisma) [inline modeles-obligatoire/theme-sujet-draft.html:1]: Should be deleted. Will file issue in the future

File: modeles-recommandes/pages-description-programme.html
Comment (delisma) [inline modeles-recommandes/pages-description-programme.html:1]: Revert changes to remove page from PR

File: modeles-recommandes/pages-directives-lois.html
Comment (delisma) [inline modeles-recommandes/pages-directives-lois.html:1]: Revert changes to remove page from PR

File: modeles-recommandes/pages-lancement-service.html
Comment (delisma) [inline modeles-recommandes/pages-lancement-service.html:1]: Revert changes to remove page from PR

File: test.html
Comment (delisma) [inline test.html:1]: Revert changes to remove page from PR

**Review by delisma — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
 
-Afin de passer à un système de conception plus souple, le plan consiste à «découpler» les spécifications détaillées des modèles des «documents de politique» officiels - la [spécification d'architecture de contenu et d'information de Canada.ca](https://www.canada.ca/fr/secretariat-conseil-tresor/services/communications-gouvernementales/specifications-contenu-architecture-information-canada.html) et le [guide de rédaction du contenu du site Canada.ca](https://www.canada.ca/fr/secretariat-conseil-tresor/services/communications-gouvernementales/guide-redaction-contenu-canada.html).
+Afin de passer à un système de conception plus souple, le plan consiste à «découpler» les spécifications détaillées des modèles des «documents de politique» officiels - la [spécification d'architecture de contenu et d'information de Canada.ca]({{ site.url }}/architecture/specifications-contenu-architecture-information-canada.html) et le [guide de rédaction du contenu du site Canada.ca](https://www.canada.ca/fr/secretariat-conseil-tresor/services/communications-gouvernementales/guide-redaction-contenu-canada.html).
```
[the remark this answers is not in the document]
Comment (prycrane) [reply README.md:5]: Done

File: architecture/utilisation-concept-canadaca.md (hunk 1 above)
Comment (prycrane) [reply architecture/utilisation-concept-canadaca.md:9]: Can you verify, @delisma ?

File: configurations-conception-communes/widget-medias-sociaux.html
Comment (prycrane) [reply configurations-conception-communes/widget-medias-sociaux.html:1]: Done

File: modeles-obligatoire/pages-publics-cibles.html
Comment (prycrane) [reply modeles-obligatoire/pages-publics-cibles.html:1]: Done

File: modeles-obligatoire/theme-sujet-draft.html
Comment (prycrane) [reply modeles-obligatoire/theme-sujet-draft.html:1]: https://github.com/canada-ca/systeme-conception/pull/137

File: modeles-recommandes/pages-description-programme.html
Comment (prycrane) [reply modeles-recommandes/pages-description-programme.html:1]: done

File: modeles-recommandes/pages-lancement-service.html
Comment (prycrane) [reply modeles-recommandes/pages-lancement-service.html:1]: Done

File: architecture/organiser-contenu.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+        <li><strong>Recherche :</strong> prend en charge la repérabilité pour la navigation thématique et institutionnelle par le biais des moteurs de recherche internes et externes</li>
+      </ul>
+      <p>La plupart des gens à la recherche de services et de renseignements gouvernementaux ne comprennent pas comment le gouvernement est organisé. Pour résoudre ce problème, la navigation par thème est le chemin de navigation prioritaire sur Canada.ca.</p>
+      <p>Voyez où se situe votre contenu dans les thèmes et les sujets correspondants de <a href="/fr/gouvernement/a-propos/systeme-conception/arborescence-thematique-types-contenu.html#arborescence">l'arborescence thématique de Canada.ca</a>.</p>
+      <figure class="mrgn-bttm-lg">
+        <figcaption class="text-center"><b>Aperçu conceptuel de la navigation de l’utilisateur</b></figcaption>
+        <img src="/content/dam/tbs-sct/images/government-communications/canada-content-style-guide/conceptual-overview-fra.png" class="img-responsive center-block" alt="Diagramme de l’aperçu conceptuel de l’architecture de l’information. La version textuelle se trouve ci-dessous&nbsp;:">
```
Comment (delisma) [inline architecture/organiser-contenu.md:49]:
```suggestion
        <img src="/content/dam/tbs-sct/images/government-communications/canada-content-style-guide/conceptual-overview-fra.png" class="img-responsive center-block" alt="Diagramme de l’aperçu conceptuel de l’architecture de l’information. La version textuelle se trouve ci-dessous&nbsp;:">
```
Change the dam thing.

**Review by delisma — CHANGES_REQUESTED** [review]

File: architecture/organiser-contenu.md [hunk 5]
```diff
@@ -0,0 +1,283 @@
+---
+altLangPage: "https://design.canada.ca/architecture/organizing-content.html"
+breadcrumbs:
+  - title: À propos de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos.html"
+  - title: Système de conception de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html"
+  - title: Spécifications du contenu et de l'architecture de l'information pour Canada.ca
+    link: "{{ site.url }}/architecture/specifications-contenu-architecture-information-canada.html"
```
[the remark this answers is not in the document]
Comment (delisma) [reply architecture/organiser-contenu.md:9]: Still need to remove the quotes in the Frontmatter.

File: architecture/elements-obligatoires.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
+---
+altLangPage: "https://design.canada.ca/architecture/mandatory-elements.html"
+breadcrumbs:
+  - title: À propos de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos.html"
+  - title: Système de conception de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html"
+  - title: Spécifications du contenu et de l'architecture de l'information pour Canada.ca
+    link: "{{ site.url }}/architecture/specifications-contenu-architecture-information-canada.html"
```
[the remark this answers is not in the document]
Comment (delisma) [reply architecture/elements-obligatoires.md:9]: Still need to remove the quotes

File: architecture/modeles.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
+---
+altLangPage: "https://design.canada.ca/architecture/templates.html"
+breadcrumbs:
+  - title: À propos de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos.html"
+  - title: Système de conception de Canada.ca
+    link: "https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html"
+  - title: Spécifications du contenu et de l'architecture de l'information pour Canada.ca
+    link: "{{ site.url }}/architecture/specifications-contenu-architecture-information-canada.html"
```
Comment (delisma) [inline architecture/modeles.md:9]: Need to remove the quotes

File: architecture/utilisation-concept-canadaca.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (delisma) [reply architecture/utilisation-concept-canadaca.md:9]: Still need to remove quotes

**Review by delisma — CHANGES_REQUESTED** [review]
A few couple of changes are still required. Almost there.

**Review by delisma — CHANGES_REQUESTED** [review]
A few couple of changes are still required. Almost there.
The images are not included in this PR and we have 22 html files compared to 24 html in the English PR.
We should consider moving the breadcrumbs for the architecture folder into the _config.yml file as a default setting.

File: architecture/elements-obligatoires.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (prycrane) [reply architecture/elements-obligatoires.md]: image
It does not seem to removing quotes.  Alas.

File: architecture/organiser-contenu.md (hunk 4 above)
Comment (prycrane) [reply architecture/organiser-contenu.md]: fixed

File: architecture/elements-obligatoires.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (prycrane) [reply architecture/elements-obligatoires.md]: I updated the breadcrumb front matter (Including on the English side.)  @delisma - please have a look at upgrading the config file.

**Review by delisma — APPROVED** [review]
LGTM

File: modeles-obligatoire/pages-publics-cibles.html
Comment (delisma) [inline modeles-obligatoire/pages-publics-cibles.html:1]: This modification wasn't required in English

**Pull request merged**